std/range/package.d

```diff
@@ -1936,23 +1953,13 @@ if (isInputRange!(Unqual!Range) &&
         }
     }
 
-    // Nonstandard
+    /// Nonstandard
```

Comment:
You're going to want something a little more descriptive here. What about this is non-standard and why does that matter?